mp4parse/src/lib.rs

```diff
@@ -35,7 +35,7 @@ use std::io::{Read, Take};
 mod macros;
 
 mod boxes;
-use boxes::{BoxType, FourCC};
+use crate::boxes::{BoxType, FourCC};
```

Comment:
This seems to work fine without the change. Is adding `crate::` a stylistic preference?

Reply:
That's what `cargo fix` does. I presume it's for consistency, because `crate::` works anywhere, and without it it only works in the same file.